autogen.sh

```diff
@@ -27,6 +27,5 @@ aclocal -Wall --install -I m4 &&
   libtoolize -Wall --copy &&
   autoheader -Wall &&
   autoconf -Wall &&
-  automake -Wall --add-missing --copy &&
-  ./configure "$@"
```

Comment:
I'll have a proper look at this tomorrow, but there's a CI failure -- if you click all the way through, the output (eventually) shows: 

```
2019-02-06T06:04:41.1107537Z  ---> 16f1d0294218
2019-02-06T06:04:41.1161300Z Successfully built 16f1d0294218
2019-02-06T06:04:41.3748100Z Successfully tagged github/projfs-dev-linux:latest
2019-02-06T06:04:41.3790098Z + ./projfs develop configure
2019-02-06T06:04:41.4468376Z >> docker run -u 1001 --pid=host --rm --name projfs-develop --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined -v /home/vsts/work/1/s/docker/..:/data/projfs github/projfs-dev-linux ./autogen.sh --enable-vfs-api
2019-02-06T06:04:43.5419691Z aclocal: installing 'm4/libtool.m4' from '/usr/share/aclocal/libtool.m4'
2019-02-06T06:04:43.5440766Z aclocal: installing 'm4/ltoptions.m4' from '/usr/share/aclocal/ltoptions.m4'
2019-02-06T06:04:43.5459454Z aclocal: installing 'm4/ltsugar.m4' from '/usr/share/aclocal/ltsugar.m4'
2019-02-06T06:04:43.5478143Z aclocal: installing 'm4/ltversion.m4' from '/usr/share/aclocal/ltversion.m4'
2019-02-06T06:04:43.5495297Z aclocal: installing 'm4/lt~obsolete.m4' from '/usr/share/aclocal/lt~obsolete.m4'
2019-02-06T06:04:44.7793676Z libtoolize: putting auxiliary files in '.'.
2019-02-06T06:04:44.7794539Z libtoolize: copying file './ltmain.sh'
2019-02-06T06:04:45.8781616Z configure.ac:26: installing './compile'
2019-02-06T06:04:45.8799271Z configure.ac:26: installing './config.guess'
2019-02-06T06:04:45.8818360Z configure.ac:26: installing './config.sub'
2019-02-06T06:04:45.8836120Z configure.ac:25: installing './install-sh'
2019-02-06T06:04:45.8857452Z configure.ac:25: installing './missing'
2019-02-06T06:04:45.9519786Z lib/Makefile.am: installing './depcomp'
2019-02-06T06:04:46.8374819Z + ./projfs develop make
2019-02-06T06:04:46.9022090Z >> docker run -u 1001 --pid=host --rm --name projfs-develop --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined -v /home/vsts/work/1/s/docker/..:/data/projfs github/projfs-dev-linux make
2019-02-06T06:04:47.6277828Z make: *** No targets specified and no makefile found.  Stop.
2019-02-06T06:04:48.0954949Z ./projfs:84:in `system': failed to run 'docker run -u 1001 --pid=host --rm --name projfs-develop --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined -v /home/vsts/work/1/s/docker/..:/data/projfs github/projfs-dev-linux make (RuntimeError)
2019-02-06T06:04:48.0956045Z 	from /home/vsts/work/1/s/docker/project.rb:71:in `run'
2019-02-06T06:04:48.0956504Z 	from /home/vsts/work/1/s/docker/project.rb:50:in `block in command'
2019-02-06T06:04:48.0956919Z 	from /home/vsts/work/1/s/docker/project.rb:49:in `each'
2019-02-06T06:04:48.0957303Z 	from /home/vsts/work/1/s/docker/project.rb:49:in `command'
2019-02-06T06:04:48.0957676Z 	from ./projfs:182:in `<main>'
```

That'd be because we no longer call `configure` from `autogen.sh`. An adjustment to the `projfs` tool is needed to restore things; right now `./projfs develop configure` runs autogen.sh only; it'll need to run autogen then configure, I assume.

Reply:
Awfully nice to have the CI looking out for this kind of thing!  Kudos and thanks @kivikakk!!